nav2_mppi_controller/CMakeLists.txt

```diff
@@ -14,9 +14,11 @@ set(XTENSOR_USE_XSIMD 1)
 find_package(ament_cmake REQUIRED)
 find_package(xsimd REQUIRED)
 find_package(xtensor REQUIRED)
```

Comment:
Note, if removing xtensor, we should remove it completely so we don't find_package, link, or in package.xml :-) 

Maybe not to change right now while still migrating, but making the note for the next steps

Reply:
Yes, I will remove them once the complete migration is done.